app/pubspec.yaml

```
@@ -23,6 +23,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  cloud_firestore: ^0.16.0+1
+  connectivity: ^2.0.2
```

Not that you should remove this, but it should be removed ASAP when we no longer use connectivity (which should be really soon).
```suggestion
  // TODO: Remove this immediately
  connectivity: ^2.0.2
```

---

will do so once everything is completed